xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -901,13 +901,13 @@ int CPVRGUIInfo::GetDuration(void) const
 int CPVRGUIInfo::GetStartTime(void) const
 {
   CSingleLock lock(m_critSection);
-  if (m_playingEpgTag)
+  if (m_playingEpgTag || m_iTimeshiftStartTime)
```

Comment:
if GUIInfo has any influence on behaviour, things wend wrong and need to be corrected differently